src/data/markdown/translated-guides/en/03 Using k6 browser/02 Running browser tests.md

```diff
@@ -25,55 +25,69 @@ To run a simple local script:
+  import { browser } from 'k6/experimental/browser';
+
+  export const options = {
+    scenarios: {
+      ui: {
+        executor: 'shared-iterations',
+        options: {
+          browser: {
+            type: 'chromium',
+          },
+        },
+      },
+    },
+    thresholds: {
+      checks: ["rate==1.0"]
+    }
+  }
 
   export default async function () {
-    const browser = chromium.launch({
-      headless: false,
-      timeout: '60s', // Or whatever time you want to define
-    });
-    const page = browser.newPage();
+    const context = browser.newContext();
+    const page = context.newPage();
 
     try {
       await page.goto('https://test.k6.io/');
       page.screenshot({ path: 'screenshot.png' });
     } finally {
       page.close();
-      browser.close();
     }
   }
   ```
 
   </CodeGroup>
 
-  The preceding code imports the `chromium` [BrowserType](/javascript-api/k6-experimental/browser/browsertype/) (currently the only available `BrowserType` implementation), and uses its `launch` method to start up a Chromium [Browser](/javascript-api/k6-experimental/browser/) process. Two parameters are passed to it. One is the `headless` parameter with the value `false` so you can see the browser launching, and `timeout` parameter with the value `60s` which will be the timeout used for various actions and navigation. For a full list of parameters that you can pass, check out the documentation for [BrowserType.launch()](/javascript-api/k6-experimental/browser/browsertype/launch/).
-  
-  After it starts, you can interact with it using the [browser-level APIs](/javascript-api/k6-experimental/browser/#browser-level-apis). This example visits a test URL and takes a screenshot of the page. Afterwards, it closes the page and the browser.
+  The preceding code imports the `browser` ([BrowserType](/javascript-api/k6-experimental/browser/browsertype/)), and uses its `newContext` method to starts a new Chromium [BrowserContext](/javascript-api/k6-experimental/browser-context/), which is an incognito window. 
+    
+  After the browser starts, you can interact with it using the [browser-level APIs](/javascript-api/k6-experimental/browser/#browser-level-apis). This example visits a test URL and takes a screenshot of the page. Afterwards, it closes the page. The browser will be closed automatically when the test run finishes.
 
   <Blockquote mod="note" title="">
 
   To provide rough compatibility with the Playwright API, the browser module API is also being converted from synchronous to asynchronous. `page.goto()` is now asynchronous so `await` keyword is used to deal with the asynchronous nature of the operation.
 
   </Blockquote>
 
-2. Then, run the test on your terminal with this command:
+1. Then, run the test on your terminal with this command:
 
 <CodeGroup labels={["Bash", "Windows: CMD", "Windows: PowerShell"]} lineNumbers={[false]}>
 
 ```bash
-$ K6_BROWSER_ENABLED=true k6 run script.js
+$ k6 run script.js
```

Comment:
Perhaps the run commands should now include some examples on how to set the new env var options (`K6_BROWSER_HEADLESS` and `K6_BROWSER_TIMEOUT`) at least.

Reply:
We can add another example here or link it to the browser API documentation. Since the API documentation will have these examples. WDYT?

Reply:
This will be resolved in https://github.com/grafana/k6-docs/pull/1222.